pages/docs/viewer.mdx

```diff
@@ -320,3 +332,39 @@ return (
   />
 );
 ```
+
+---
+
+### Custom displays
```

Comment:
```suggestion
### Custom canvas displays
```

I wonder if we should use the word `canvas` or `Canvas` here to speak towards a IIIFy way of thinking? The canvas is what the annotation resource is _painted_ on.

Reply:
Yea, I like being specific here.  Good call.